src/data/bool/basic.lean

```diff
@@ -225,4 +225,8 @@ by cases b; simp only [of_nat,to_nat]; exact dec_trivial
 ⟨λ Hinj Heq, ff_ne_tt (Hinj Heq),
   λ H x y hxy, by { cases x; cases y, exacts [rfl, (H hxy).elim, (H hxy.symm).elim, rfl] }⟩
 
+/-- aka. Kaminski's Equation -/
```

Comment:
Named theorems should be bolded per [our documentation guidelines](https://leanprover-community.github.io/contribute/doc.html#doc-strings):
```suggestion
/-- **Kaminski's Equation** -/
```